java/fundamentals/intro-to-java/java-versions.md

```diff
@@ -32,16 +32,28 @@ Transitioning from one version to another has always left room for problems. Usu
 
 One way to avoid this from happening is stopping public support for older versions, which has already happened. As of 2015, *Java 8* is the only publicly supported version.
 
+Update: 
```

Comment:
I think it's fine to add this information without the "Update" field. It's not different from any other insight change so there's no need to explicitly differentiate added text from existing